templates/pwm.asg.element.json

```diff
@@ -161,12 +161,6 @@
                                     "Fn::Join": [
                                         "",
                                         [
-                                            "arn:aws:logs:*:",
-                                            {
-                                                "Ref": "AWS::AccountId"
-                                            },
-                                            "log-group:/aws/ec2/lx/${AWS::StackName}:*",
```

Comment:
This wasn't working, because this isn't a valid way to reference `${AWS::StackName}` in Fn::Join...